package.json

```diff
@@ -25,7 +25,7 @@
     "url": "https://github.com/ahmadnassri/mkdirp-promise/issues"
   },
   "scripts": {
-    "compile": "babel -q src",
+    "compile": "babel -q src --out-dir lib",
```

Comment:
note how travis is using this:

https://github.com/ahmadnassri/mkdirp-promise/blob/master/.travis.yml#L20-L23